util/modlog.js

```diff
@@ -58,46 +58,32 @@ module.exports = class ModLog {
             .setAuthor(this.moderator.tag, this.moderator.avatar)
             .setColor(ModLog.colour(this.type))
             .setDescription([
-                `**Type**: ${this.type}`,
+                `**Type**: ${this.type[0].toUpperCase() + this.type.slice(1)}`,
                 `**User**: ${this.user.tag} (${this.user.id})`,
-                `**Reason**: ${this.reason || `Use \`${this.guild.configs.prefix}reason ${this.case} to claim this log.\``}`
+                `**Reason**: ${this.reason || `Use \`${this.guild.configs.prefix}reason ${this.case}\` to claim this log.`}`
             ])
             .setFooter(`Case ${this.case}`)
             .setTimestamp();
         return embed;
     }
 
-    // Here we get the case number and create a modlog provider entry
-
     async getCase() {
-        const modlogs = await this.provider.get('modlogs', this.guild.id);
-        if (!modlogs) return this.provider.create('modlogs', this.guild.id, [this.pack]).then(() => 0);
-        modlogs.push(this.pack);
-        await this.provider.replace('modlogs', this.guild.id, modlogs);
-        return modlogs.length - 1;
+        this.case = this.guild.configs.modlogs.length;
+        await this.guild.configs.update('modlogs', this.pack);

```

Comment:
return its already a promise

Reply:
so just straight up `return` instead of `await` ?